unreal_modloader/src/app.rs

```diff
@@ -379,7 +398,7 @@ impl ModLoaderApp {
                         row.col(|ui| {
                             ui.push_id(mod_id, |ui| {
                                 // becasue ComboBox .chnaged doesn't seem to work
```

Comment:
I think this was fixed, could you implement it?

Reply:
not sure how to do this so I'll leave it up to you